README.md

```diff
@@ -1,5 +1,5 @@
 # Tsukihime-Translation
-Open source translation for the Tsukihime Remake game prepared by [Clovermoon](https://twitter.com/CloverMoonTL) and [Tsukihimates](https://twitter.com/tsukihimates).
+Open source translation for the Tsukihime Remake game prepared by [Tsukihimates](https://twitter.com/tsukihimates).
```

Comment:
Open source translation for Tsukihime -A Piece of Blue Glass Moon- prepared by [Tsukihimates](https://twitter.com/tsukihimates).